### DISCUSSION TITLE
How to get fields of an particular category articles into modules..

### DISCUSSION THREAD

**Discussion by amolsawant1989** [joomla/joomla-cms#35566] [Q&A]
Hello community,

as i need help in calling an article all fields including custom fields into my module.. also those article will be from specific category.
how to access / call/ render those article fields into module.. please help

**Comment by alexandreelise** [comment]
Hello @amolsawant1989 , I think that might be what you are looking for [Custom Fields inside Module](https://gist.github.com/alexandreelise/e04d417c9f911ce2ab2a3e931142e89b)

**Reply by amolsawant1989** [reply]
Thanks but getting error after putting code inside module ...

**Warning : Undefined variable $list** and **Warning: foreach() argument must be of type array|object, null given**

Kindly Help

**Reply by alexandreelise** [reply]
Hello @amolsawant1989 as the snippet of code suggests, it works for joomla core latest news and normally should work with any joomla core modules involving articles. If your module is custom I cannot help you without seeing the code. This error tells your that $list variable that should contain the list of articles coming from you module helper.php file is not defined or does not exists.
